TriforkSwiftExtensions/TriforkSwiftExtensions/Data/DataExtension.swift

```diff
@@ -11,12 +11,12 @@ import Foundation
 public extension Data {
     
     /// Converts to string if possible, otherwise it returns `nil`
-    public func toString() -> String? {
```

Comment:
Xcode 10.2 says it is redundant 🤷‍♂️

Reply:
Isn't just because it doesn't understand that it's a framework? The Swift 5 description still sounds like it is necessary with public 😬

> Open access and public access enable entities to be used within any source file from their defining module, and also in a source file from another module that imports the defining module. You typically use open or public access when specifying the public interface to a framework. The difference between open and public access is described below.
> Internal access enables entities to be used within any source file from their defining module, but not in any source file outside of that module. You typically use internal access when defining an app’s or a framework’s internal structure.

Reply:
It generates a lot of annoying warnings... 
I think we should test if we can remove the `public` on extension or the method

Reply:
Yeah. So do you think the `public extension` part overrides the default access level on everything within it? I can't find anything on this in the release notes on swift 5.